### DISCUSSION TITLE
Add pornhub or other porn sites extension please please please please please please please

### DISCUSSION THREAD

**Discussion by Abubakarch768** [aniyomiorg/aniyomi#812] [Ideas]
Add pornhub or other porn sites extension please please please please please please please such as xvideos. Porntrex or xhamster.

**Comment by FrenchPrick** [comment]
Down bad much 🤣

**Comment by ChrolloLucilfe** [comment]
bro down bad ☠️😭

**Comment by 12ohit** [comment]
> Add pornhub or other porn sites extension please please please please please please please such as xvideos. Porntrex or xhamster.

https://github.com/jmir1/unofficial-aniyomi-extensions/tree/repo/apk

**Reply by Solovertical** [reply]
It need update
